R/bambu-processReads_scoreReadClasses.R

```diff
@@ -2,17 +2,19 @@
 #' @param se summerized experiment object with read classes/ranges
 #' @param genomeSequence genomeSequence
 #' @param annotations GRangesList of annotations
-scoreReadClasses = function(se, genomeSequence, annotations, 
-                                    min.readCount = 2, verbose = FALSE){
+scoreReadClasses = function(se, genomeSequence, annotations, defaultModels, 
+                            fit = TRUE, min.readCount = 2, verbose = FALSE){
     start.ptm <- proc.time()
     options(scipen = 999) #maintain numeric basepair locations not sci.notfi.
+    if (is.character(se)) 
+        se <- readRDS(file = se)
```

Comment:
might not be required, in the current code se will always be a summarizedExperiment object

Reply:
done in 41b06efeb255d97fbe70af4b66bb535b31fa4723